src/main/resources/feedbackQuestionConstSumEditFormTemplate.html

```diff
@@ -12,7 +12,7 @@
             </div>
         </div>
 
-        <input type="hidden" name="${Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}"
+        <input type="hidden" class="nonDestructive" name="${Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}"
```

Comment:
`nonDestructive` is actually a legacy class, I didn't create it. To change this, I'll have to look where else it is used. Shall I proceed with the change?